### ISSUE TITLE
"I Agree" button is not taken into account

### ISSUE THREAD

**Issue by nametable** [nametable/sau-daily-survey-bot#4]
A proper request (although the current request works, should look like this)

```
PersonID=048****&
Answers.Index=2&
Answers[2].QuestionID=2&
Answers[2].Answer=No&
Answers.Index=4&
Answers[4].QuestionID=4&
Answers[4].Answer=No&
Answers.Index=7&
Answers[7].QuestionID=7&
Answers[7].Answer=No&
Answers.Index=5&
Answers[5].QuestionID=5&
Answers[5].Answer=No&
Answers.Index=6&
Answers[6].QuestionID=6&
Answers[6].Answer=I+Agree
```
Currently, all questions are being answered with `no`, which is not correct now that there is an "I Agree" button.

**Comment by nametable** [comment]
The radio in the form looks like this
```html
<input id="op-agree-6" data-val="true" data-val-required="You need to agree to proceed." name="Answers[6].Answer" type="radio" value="I Agree">
```
Therefore a simple fix should be checking if the `value` of the input is "I Agree", and if it is, return the answer "I+Agree".

**Comment by nametable** [comment]
Fixed in 15175c3
